charts/orchestrator/Chart.yaml

```diff
@@ -1,8 +1,7 @@
 apiVersion: v2
 name: orchestrator
 description: >
-  Helm chart to deploy the Orchestrator solution suite on OpenShift, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,  Knative Eventing, Knative Serving, Data Index and Job Service.
-
+  Helm chart to deploy the Orchestrator solution suite on OpenShift, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,  Knative Eventing, Knative Serving, Data Index and Job Service. #magic___^_^___line
```

Comment:
Do we need to keep `Data Index and Job Service`? as far as I understand now it is deployed by sonataFlow operator

Reply:
Updated.